content/bn/event-driven-architecture.md

```diff
@@ -0,0 +1,26 @@
+---
+title: ঘটনা-চালিত স্থাপত্য
+status: Completed
+category: ধারণা
+tags: ["architecture", "", ""]
+---
+
+## এটা কি
+
+ঘটনা চালিত স্থাপত্য হল একটি সফ্টওয়্যার স্থাপত্য যা ঘটনা তৈরি, প্রক্রিয়াকরণ এবং ব্যবহারকে প্রচার করে।
+একটি ঘটনা হল একটি অ্যাপ্লিকেশনের অবস্থার পরিবর্তন।
+উদাহরণস্বরূপ, একটি রাইড-শেয়ারিং অ্যাপে রাইডের প্রশংসা করা একটি ঘটনার প্রতিনিধিত্ব করে৷
+এই স্থাপত্যটি এমন কাঠামো তৈরি করে যেখানে ঘটনাগুলো তাদের উত্স থেকে (অ্যাপটি একটি রাইডের অনুরোধ করে) থেকে পছন্দসই রিসিভারগুলিতে (আশেপাশে উপলব্ধ ড্রাইভারদের অ্যাপগুলি) সঠিকভাবে রুট করা যেতে পারে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+যত বেশি তথ্য বাস্তব সময় পরিণত হয়, ঘটনাগুলো ক্যাপচার করা এবং উপযুক্ত [সার্ভিস](/service/) যা ঘটনার অনুরোধগুলি প্রক্রিয়া করতে হবে তা নিশ্চিত করার জন্য নির্ভরযোগ্য উপায়গুলি খুঁজে পাওয়া ক্রমশ চ্যালেঞ্জিং হয়ে উঠছে৷
```

Comment:
Kindly make this a more suitable sentence "যত বেশি তথ্য বাস্তব সময় পরিণত হয়, ঘটনাগুলো ক্যাপচার করা এবং উপযুক্ত [সার্ভিস](/bn/service/) যা ঘটনার অনুরোধগুলি প্রক্রিয়া করতে হবে তা নিশ্চিত করার জন্য নির্ভরযোগ্য উপায়গুলি খুঁজে পাওয়া ক্রমশ চ্যালেঞ্জিং হয়ে উঠছে৷"